.travis.yml

```diff
@@ -1,8 +1,14 @@
 language: node_js
 node_js:
 - node
-services: mongodb
+services: postgres
 script:
   - npm test -- --coverage
+env:
+  global:
+    - FILE="__test__/postgres.mock.sql"
+    - TEST_DB_FILE="`pwd`/$FILE"
```

Comment:
I see that you set the global env for Travis, nice! Would you like to share with me what this line does though? I'm very curious.

Reply:
@phucciTr absolutely! These lines create a new `env` variable with the path to the `postgres.sql` file. If you look on lines 11 and 12, that `env` variable is then used to run that `postgres.sql` file to create the test database I need within the Travis testing environment so that there is a db and table to insert the test data/validate from.